DllImportGenerator/DllImportGenerator/StubCodeGenerator.cs

```diff
@@ -235,11 +235,41 @@ public override (string managed, string native) GetIdentifiers(TypePositionInfo
                                 invoke));
                     }
 
+                    if (this.dllImportData.SetLastError)
+                    {
+                        // Marshal.SetLastSystemError(0);
```

Comment:
Let's make a readonly constant for the official "success". We should comment that it is `S_OK` per `HRESULT` rules and `0` per POSIX.